### ISSUE TITLE
Not working on version 301269

### ISSUE THREAD

**Issue by Nevaehni** [rauenzi/BDPluginLibrary#223]
TypeError: Cannot read properties of undefined (reading 'prototype')
    at ./src/ui/colorpicker.js (betterdiscord://plugins/0PluginLibrary.plugin.js:3845:56)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/ui/settings/types/color.js (betterdiscord://plugins/0PluginLibrary.plugin.js:5147:70)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/ui/settings/index.js (betterdiscord://plugins/0PluginLibrary.plugin.js:4792:70)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/ui/ui.js (betterdiscord://plugins/0PluginLibrary.plugin.js:6125:67)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/modules/pluginupdater.js (betterdiscord://plugins/0PluginLibrary.plugin.js:1712:60)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/structs/plugin.js (betterdiscord://plugins/0PluginLibrary.plugin.js:3585:80)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/structs/structs.js (betterdiscord://plugins/0PluginLibrary.plugin.js:3816:65)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/modules/domtools.js (betterdiscord://plugins/0PluginLibrary.plugin.js:656:65)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at ./src/modules/modules.js (betterdiscord://plugins/0PluginLibrary.plugin.js:1567:67)
    at __webpack_require__ (betterdiscord://plugins/0PluginLibrary.plugin.js:6168:41)
    at eval (betterdiscord://plugins/0PluginLibrary.plugin.js:6215:65)
    at eval (betterdiscord://plugins/0PluginLibrary.plugin.js:6322:3)
    at eval (betterdiscord://plugins/0PluginLibrary.plugin.js:6325:12)
    at N.requireAddon (betterdiscord/renderer.js:5:28792)
    at N.loadAddon (betterdiscord/renderer.js:5:6412)
    at N.loadAddon (betterdiscord/renderer.js:5:26460)
    at N.reloadAddon (betterdiscord/renderer.js:5:7302)
    at betterdiscord/renderer.js:5:4426
    
    
 ```md
## Discord Info
stable 301269 (6e5b8c9) Host 1.0.9149 x64 (48589) Build Override: N/A Windows 11 64-bit (10.0.22631)

## BetterDiscord
stable 1.10.1

### Plugins (6 Enabled):
- CallTimeCounter (Enabled)
- DoNotTrack (Enabled)
- FileViewer (Enabled)
- removeTrackingURL (Enabled)
- SplitLargeMessages (Enabled)
- Uncompressed Images (Enabled)
- YABDP4Nitro
- ZeresPluginLibrary

### Themes (0 Enabled):

```

**Comment by TrapStoner** [comment]
Im on stable 301269 (6e5b8c9) Host 1.0.9148 x64 (48468) and i have it too

**Comment by rauenzi** [comment]
Duplicate of #221

**Comment by Nevaehni** [comment]
Can confirm, this has been fixed now with v2.0.20.
